websites/A/AnkiWeb/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "author": {
+    "id": "193714715631812608",
+    "name": "theusaf"
+  },
+  "service": "AnkiWeb",
+  "description": {
+    "en": " Anki is a program which makes remembering things easy. Because it's a lot more efficient than traditional study methods, you can either greatly decrease your time spent studying, or greatly increase the amount you learn."
```

Comment:
```suggestion
    "en": "Anki is a program which makes remembering things easy. Because it's a lot more efficient than traditional study methods, you can either greatly decrease your time spent studying, or greatly increase the amount you learn."
```